tools/internal_ci/linux/grpc_full_performance_upstream.cfg

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017 gRPC authors.
```

Comment:
nit: not sure about the _upstream suffix (we've been using "master" for that meaning so far I think), but leaving up to you.

Reply:
Switched to master. I think it makes more sense, especially if we use "release" as our other suffix.